src/inmanta/module.py

```diff
@@ -2112,8 +2115,11 @@ def load(self, install: bool = False) -> None:
                 self.install_modules()
             self.get_complete_ast()
             self.loaded = True
+            start = time()
             self.verify()
             self.load_plugins()
+            end = time()
+            LOGGER.debug("Module loading took %0.03f seconds", end - start)
```

Comment:
Shouldn't this be:
```suggestion
            LOGGER.debug("Plugin loading took %0.03f seconds", end - start)
```
Why do we include the time in the call to `verify()` in the measurement?

Reply:
1. I think it is not sufficiently important to get its own entry
2. It is conceptually close to plugin loading (it is purely about version number)
3. I want to the sum of the measured number to be close to the total time